tests/lib/ignored-paths.js

```diff
@@ -164,6 +164,20 @@ describe("IgnoredPaths", () => {
                 )
             );
         });
+
+        it("should load package.json's eslintIgnore files if no specified ignore file", () => {
+            const ignoredPaths = new IgnoredPaths({ ignore: true, cwd: getFixturePath("package-json-ignore") });
+
+            assert.isTrue(ignoredPaths.contains("hello.js"));
+            assert.isTrue(ignoredPaths.contains("world.js"));
+        });
+
+        it("should not load package.json's eslintIgnore files if specified ignore file", () => {
```

Comment:
Nitpick: the names of these tests were a bit confusing to me at first. Would you mind rewording them?

* I wasn't sure what "load" means in this context (since eslint as a whole would explicitly *not* load the files that were ignored). It might be better to say "should *use* package.json's eslintIgnore files".
* Can you say ".eslintignore file" rather than "ignore file"? I wasn't sure whether "ignore file" referred to a file being ignored or the .eslintignore file itself.